cluster-autoscaler/cloudprovider/azure/azure_instance_types.go

```diff
@@ -16,15 +16,16 @@ limitations under the License.
 
 package azure
 
-type instanceType struct {
+// InstanceType is the sepc of Azure instance
+type InstanceType struct {
 	InstanceType string
 	VCPU         int64
 	MemoryMb     int64
 	GPU          int64
 }
 
 // InstanceTypes is a map of azure resources
-var InstanceTypes = map[string]*instanceType{
+var InstanceTypes = map[string]*InstanceType{
```

Comment:
could you update the generated instance types together?

Reply:
what do you mean?

Reply:
sure